panoptes/routes/api.py

```diff
@@ -66,3 +66,14 @@ def get_job_of_workflow(workflow_id, job_id):
         return jsonify({'msg': 'Workflow not found',
                         'jobs': [],
                         'count': 0})
+
+
+@routes.route('/api/delete/<workflow_id>', methods=['GET'])
+def set_db_delete(workflow_id):
+    delete=del_db_wf(workflow_id)
+    if delete:
+        return jsonify({'msg': "Delete Complete Correctly ",'Workflow': workflow_id})
+    else:
+        return jsonify({ "error": {
```

Comment:
Maybe change it to something like this:

{'error': 404, 'msg': 'Unable to delete Workflow 1. Please check if workflow 1 exists'}